### PR TITLE
Add wrapper projects of various frameworks

### PR DESCRIPTION
**Pull request by Matsuuu**

### CODE REVIEW

**Comment by pshihn** [comment]
This looks good. Why have two WC projects?
Can you pick one that you prefer. I think both are yours.

**Comment by Matsuuu** [comment]
I made the lit one first, and then decided that this didn't need such level of complexity. 
I think I'll rebrand the vanilla version as `wc-rough-notation` or just `<rough-notation>`, and will update here with the updated list :) 
Will do it tomorrow and mention it here!

**Comment by Matsuuu** [comment]
Hey @pshihn . 

I removed the lit element one from the list, and updated the vanilla WC one to use tags `<rough-notation` and `<rough-notation-group>` .

This PR is mergeable now, if you approve of the collection of wrappers

File: README.md [hunk 1]
```diff
@@ -109,3 +109,15 @@ Hides the annotation if showing. This is not animated.
...
 Unlinks the annotation from the element. 
+
+
+
+
+## Wrappers
+
+Rough Notation has wrappers for multiple libraries and frameworks
+
+[React Rough Notation](https://github.com/linkstrifer/react-rough-notation)
```
Comment (fspoettel) [inline README.md:120]: This is a bit hard to read, what about using a bulleted list?

- [React Rough Notation](https://github.com/linkstrifer/react-rough-notation)
- [Svelte Rough Notation](https://github.com/dimfeld/svelte-rough-notation)
It is easier to scan it at a glance.

File: README.md [hunk 2]
```diff
@@ -109,3 +109,15 @@ Hides the annotation if showing. This is not animated.
 
 #### remove()
 Unlinks the annotation from the element. 
+
+
+
+
+## Wrappers
+
+Rough Notation has wrappers for multiple libraries and frameworks
```
Comment (fspoettel) [inline README.md:118]: Missing a `:` at the end

**Comment by Matsuuu** [comment]
> Thanks for compiling this and your work on the web components wrapper! Two suggestions after looking at the rendered readme

Oh, I was a bit hasty on those ones and forgot the dashes, or my linter just dropped them for some reason.

Fixed them now @fspoettel . Thanks for spotting them out!

**Comment by pshihn** [comment]
Thanks!

**Pull request merged**